Assets/Toshima Support Center/Scripts/TutorialUIManager.cs

```diff
@@ -59,7 +59,6 @@ private enum KeyWord
 
     private void Awake()
     {
-        // 初期化用関数を呼び出す
         Initializ();
```

Comment:
```suggestion
        Initialize();
```
かなりスペルミスしています。　養いましょう、「気づき」を
プログラマとして生きていきたいなら、平均以上の英語能力は必須です

Reply:
これは、すみません。スペルミスです。

Reply:
修正内容は
ちゃんとしたスペルに修正しました。
Initializ→initializeに変更しました。